server/mailet/integration-testing/src/test/java/org/apache/james/mailets/RemoteDeliveryDKIMIntegrationTest.java

```diff
@@ -130,9 +129,13 @@ void tearDown() {
     class WhenEnable8BitMime {
         @Disabled("JAMES-3016 assertion failed:" +
             "org.apache.james.jdkim.exceptions.PermFailException: Computed bodyhash is different from the expected one")
```

Comment:
why JAMES-3016 is closed if it is not fixed?

Reply:
My bad... Reopening it then

Reply:
It's fixed, by removing `allow8bitmime` in the code base. In this test, I want to show people the issue can appear when you intentionally enable `allow8bitmime` in your James server

Reply:
oh right sorry I read to fast.
ok to merge this as is.
but for me as it is an expected behavior the test should not be disabled, it just should be negated.

Reply:
Damn I don't have the right, please help me close it